core/ir/x86/instr.c

```diff
@@ -40,8 +40,31 @@
 #include "instr.h"
 #include "decode.h"
 #include "decode_private.h"
+#include "encode_api.h"
```

Comment:
?? I did not add this. clangd issue?
Will investigate it's necessary.

Reply:
It's best practice to add encode_api.h (which defined dr_isa_mode_t) rather than relying on other headers.